tripzone/src/App.js

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import Homepage from './screens/homepage/Homepage';
+import { Provider } from './context/loginContext';
+import { AppProvider } from "./context/appContext";
+
+function App() {
+  return (
+    <Provider>
```

Comment:
could you let me know what provider it is , because it is very generic name provider